src/app/components/selectable-box/double-column/DoubleSelectableBox.jsx

```diff
@@ -5,30 +5,10 @@ import { useSort } from "../../../hooks/useSort";
 import DoubleSelectableBoxItem from "./DoubleSelectableBoxItem";
 import Sort from "../../sort";
 
-const propTypes = {
-    headings: PropTypes.array.isRequired,
-    search: PropTypes.string,
-    items: PropTypes.arrayOf(
-        PropTypes.shape({
-            id: PropTypes.string.isRequired,
-            selectableBox: PropTypes.shape({
-                firstColumn: PropTypes.string.isRequired,
-                secondColumn: PropTypes.string.isRequired,
-            }),
-        })
-    ).isRequired,
-    activeItemID: PropTypes.string,
-    handleItemClick: PropTypes.func.isRequired,
-    isUpdating: PropTypes.bool,
-};
-
 const DoubleSelectableBox = props => {
-    const [activeSort, setActiveSort] = useState({ key: "name", direction: "ASC" });
-    const { sortedItems, requestSort, sortConfig } = useSort(props.items, activeSort);
-
-    const handleSortClick = (key, direction) => {
-        setActiveSort({key, direction});
-        requestSort({key, direction});
+    const { sortedItems, requestSort, sortConfig } = useSort(props.items, { key: "name", direction: "ASC" });
+    const handleSortClick = key => {
```

Comment:
Is this redundant now? Should we just call `requestSort(...)` instead?

Reply:
good idea